### PR TITLE
Target java8 so server code can use AssessmentModel

### PR DESCRIPTION
**Pull request by nategbrown9**

### CODE REVIEW

File: assessmentResults/src/commonMain/kotlin/org/sagebionetworks/assessmentmodel/navigation/Direction.kt [hunk 1]
```diff
@@ -1,31 +1,20 @@
...
      * Move backward through the assessment.
      */
-    @SerialName("backward")
-    @JsonNames("backward", "Backward")
     Backward,
     /**
      * Exit the assessment early. If this direction indicator is set, then the entire assessment run should end.
      */
-    @SerialName("exit")
-    @JsonNames("exit", "Exit")
```
Comment (syoung-smallwisdom) [inline assessmentResults/src/commonMain/kotlin/org/sagebionetworks/assessmentmodel/navigation/Direction.kt:28]: default is lower case?

**Review by syoung-smallwisdom — APPROVED** [review]
Looks good. One question.

File: assessmentResults/src/commonMain/kotlin/org/sagebionetworks/assessmentmodel/navigation/Direction.kt (hunk 1 above)
Comment (nategbrown9) [reply assessmentResults/src/commonMain/kotlin/org/sagebionetworks/assessmentmodel/navigation/Direction.kt:28]: default is the name of the enum

Comment (syoung-smallwisdom) [reply assessmentResults/src/commonMain/kotlin/org/sagebionetworks/assessmentmodel/navigation/Direction.kt:28]: Which is capitalized and doesn't match iOS?

**Review by syoung-smallwisdom — APPROVED** [review]
looks good

**Pull request merged**